### PR TITLE
📝 make repo more opensource friendly

### PR DESCRIPTION
**Pull request by abdelhamidbakhta**
## Description

Add some documentation and useful files for external contributors to be onboarded easily.

Add some workflows to clean up inactive issues and pull requests, synchronise labels.

## Changes summary

- Update `Readme` file and add sections and badges
- Add `.github/ISSUE_TEMPLATE` directory with 3 kinds of issue templates
    - Bug report
    - Feature request
    - Codebase improvement
 - Add some Github workflows
     - `labels.yml`: Synchronize Github labels of the repository
     - `lock.yml`: Lock closed issues and PRs
     - `pr-label.yml`: Verify PR has a valid label
     - `stale.yml`: Clean up stale issues and PRs
- Add pull request template
- Add some typical open source files under `docs` directory
    - Code of conduct: Contributor Covenant Code of Conduct
    - Contributing: Guidelines for contributors
    - Security: Guidelines for Reporting a Vulnerability

### CODE REVIEW

File: docs/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If there are any vulnerabilities in **Beerus**, don't hesitate to _report them_.
```
Comment (lucidLuckylee) [inline docs/SECURITY.md:5]: This should be ZeroSync :)

**Comment by lucidLuckylee** [comment]
Readme and Security still include some Beerus related info. If these are meant to be templates I can edit them accordingly after merging.

**Comment by abdelhamidbakhta** [comment]
> Readme and Security still include some Beerus related info. If these are meant to be templates I can edit them accordingly after merging.

My bad. Will update tomorrow. Or you can edit after merge if you prefer, it’s up to you

**Comment by abdelhamidbakhta** [comment]
> Readme and Security still include some Beerus related info. If these are meant to be templates I can edit them accordingly after merging.

Fixed

**Comment by RobinLinus** [comment]
Closed with c97dbcc .

**Pull request closed without merging**